lib/Test/Ika/Reporter/JUnit.pm

```diff
@@ -0,0 +1,140 @@
+package Test::Ika::Reporter::JUnit;
+
+use strict;
+use warnings;
+use utf8;
+
+use Test::Ika::Reporter::Spec;
+
+use XML::Simple;
+use Time::HiRes qw(gettimeofday tv_interval);
+
+sub new {
+    my ($class, $args) = @_;
+
+    return bless {
+        builder    => Test::Ika::Reporter::JUnit::IO->new,
+        time       => [ gettimeofday ],
+        test_suite => {},
+        describes  => [],
+    }, $class;
+}
+
+sub builder { shift->{builder} }
+
+sub describe {
+    my ($self, $name) = @_;
+
+    push @{$self->{describes}}, $name;
+    return Scope::Guard->new(sub {
```

Comment:
Added in https://github.com/sahilsingla90/Test-Ika/commit/a2a363dffcf2ddf8d062234643bf53a275736a75